esp/scm/ws_packageprocess.ecm

```diff
@@ -51,6 +51,31 @@ ESPresponse [exceptions_inline] AddPackageResponse
     ESParray<string, File> FilesNotFound;
 };
 
+ESPrequest CopyPackageMapRequest
+{
+    string SourcePath;
+    string RemoteUserName;
+    [password] string RemotePassword;
+    string Target;
+    string Process;
+    string PMID;
+    boolean Activate;
+    string DaliIp;
+    bool GlobalScope(0);
+    string SourceProcess;
+    [min_ver("1.02")] bool PreloadAllPackages(false);
```

Comment:
It seems like this entire structure would not be available until  ver 1.02 (or 1.03 if we update the interface version)